fili-core/src/main/java/com/yahoo/bard/webservice/jobs/AsyncUtils.java

```diff
@@ -46,7 +47,7 @@ public static Response makeJobMetadataResponse(String jobMetadata) {
      * @return A PreResponse containing the error information that should be sent to the user
      */
     public static PreResponse buildErrorPreResponse(Throwable throwable) {
-        ResponseContext responseContext = new ResponseContext(Collections.emptyMap());
+        ResponseContext responseContext = new ResponseContext(new LinkedHashMap<>());
```

Comment:
Rather than make a new object each time, it would be good if we could only create 1 object and just re-use that over and over (like what `Collections.emptyMap()` did).